Plots/Panels/NCL_panel_10.py

```diff
@@ -0,0 +1,258 @@
+"""
+NCL_panel_10.py
+===============
+This script illustrates the following concepts:
+   - Drawing Hovmueller plots
+   - Attaching plots along the Y axis
+   - Using a blue-white-red color map
+   - Drawing zonal average plots
+   - Paneling attached plots
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/panel_10.ncl and https://www.ncl.ucar.edu/Applications/Scripts/panel_attach_10.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/panel_10_lg.png and https://www.ncl.ucar.edu/Applications/Images/panel_attach_10_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import numpy as np
+import xarray as xr
+import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1.inset_locator import inset_axes
+import matplotlib.gridspec as gridspec
+
+import geocat.datafiles as gdf
+import geocat.viz.util as gvutil
+from geocat.viz import cmaps as gvcmaps
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine
+# and load the data into xarrays
+filename = 'chi200_ud_smooth.nc'
+ds = xr.open_dataset(gdf.get('netcdf_files/' + filename))
+
+lon = ds.lon
+times = ds.time
+scale = 1000000
+chi = ds.CHI
+chi = chi / scale
+
+# Calculate zonal mean
+mean = chi.mean(dim='lon')
+
+###############################################################################
+# Create Single Plot:
+fig, (ax1, ax2) = plt.subplots(nrows=1,
+                               ncols=2,
+                               sharey=True,
+                               figsize=(12, 9),
+                               gridspec_kw=dict(wspace=0,
+                                                width_ratios=[0.75, 0.25],
+                                                left=0.15,
+                                                right=0.85,
+                                                top=0.85,
+                                                bottom=0.15))
+# Create inset axes for color bar
+cax1 = inset_axes(ax1,
+                  width='100%',
+                  height='7%',
+                  loc='lower left',
+                  bbox_to_anchor=(0, -0.15, 1, 1),
+                  bbox_transform=ax1.transAxes,
+                  borderpad=0)
+
+# Draw contour lines
+ax1.contour(lon,
+            times,
+            chi,
+            levels=np.arange(-12, 13, 2),
+            colors='black',
+            linestyles='solid',
+            linewidths=.5)
+
+# Draw filled contours
+cf = ax1.contourf(lon,
```

Comment:
```suggestion
ax1.contourf(lon,
```
cf is defined and unused, but contourf is used. So just removing the variable definition from this line works